app/navigation/nonsipp/TransferOutNavigator.scala

```diff
@@ -35,6 +35,9 @@ object TransferOutNavigator extends JourneyNavigator {
 
     case WhatYouWillNeedTransferOutPage(srn) =>
       controllers.routes.UnauthorisedController.onPageLoad()
+
+    case ReceivingSchemeNamePage(srn, index, transferIndex) =>
```

Comment:
@AlessandroCinque @narendrapaduchuri just make sure please you guys agree who is doing the navigation from member list to this page, so it is not missed out.